metrics.md

```diff
@@ -103,26 +100,28 @@ Notes:
 - Partial date elements are interpreted as the latest date possible when compared to birthDate (e.g., 1999 would be the equivalent of 1999-12-31)
 - Partial date elements are interpreted as the earliest date possible when compared to deceasedDateTime (e.g., 1999 would be the equivalent of 1999-01-01)
 
+Suggested parameters:
+- `threshold_days` - allowed period after death date, defaults to 30 days
+
 ### q_date_recent
 **[plausibility]** Expect Date to be in Recent Past
 
-Examine each of the standard [date elements](#by-date).
-
-Parameters:
-- `run_time` - the start datatime when the set of queries that includes this metric was run
-- `start_date` - defaults to `1950-01-01` (kind of arbitrary, but it's what OMOP is using)
-- `end_date` - defaults to metric run time plus 0 days
-
 Denominator resource inclusion:
-- The date element is populated
+- Resource is in the table of standard [date elements](#by-date).
 
 Numerator resource inclusion:
-- The date elements is before `start_date` parameter and/or after `end_date` parameter
+- Any of the populated date elements are before `start_date` parameter and/or after `end_date` parameter
+- The [status](#by-status) is not `entered-in-error`
+- For Encounters, the [status](#by-status) is not `planned`
 
 Notes:
 - Partial date elements are interpreted as the latest date possible when compared to start_date (e.g., 1999 would be the equivalent of 1999-12-31)
 - Partial date elements are interpreted as the earliest date possible when compared to end_date (e.g., 1999 would be the equivalent of 1999-01-01)
 
+Suggested parameters:
+- `start_date` - a recommended default is `1900-01-01` (arbitrary, but it allows for patients with backdated-to-birth resources, like AllergyIntolerances)
```

Comment:
Oh I also changed this recommended start date from 1950 to 1900 - at BCH we had this situation of birth-backdated allergy intolerances.